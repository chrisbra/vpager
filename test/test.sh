#!/bin/sh

#set -x

# Setup some color variables
red=$(tput setaf 1)
green=$(tput setaf 2)
reset=$(tput sgr0)

for i in */; do
  (
    cd "$i" &&
    test -f "cmd" &&
    ./cmd > test.out &&
    diff -qu test.expected  test.out >/dev/null ;
    if [ $? -ne 0 ]; then
      printf "Test %s:\t\t[${red}Failed${reset}]\n" ${i%%/}
      diff -u test.expected test.out
    else
      printf "Test %s:\t\t[${green}OK${reset}]\n" ${i%%/}
    fi
    )
done
