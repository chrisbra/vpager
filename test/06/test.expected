]51;["call", "Tapi_Vpager_setup", ["vpager_setup", "%d _","setl ft=diff","wincmd H"]]]51;["call", "Tapi_Vpager",  ["vpager", "foobar \"one two\" "]]foobar "one two" 
]51;["call", "Tapi_Vpager",  ["vpager", " 1/2"]] 1/2
]51;["call", "Tapi_Vpager",  ["vpager", " [ some more "]] [ some more 
]51;["call", "Tapi_Vpager",  ["vpager", " input \u001b \u001b \\ ] "]] input   \ ] 
